ADDING_A_PLUGIN.md

```diff
@@ -54,7 +54,7 @@ You can start building the backend by subclassing `TBPlugin` in [`base_plugin.py
 
 ```python
 class MyPlugin(base_plugin.TBPlugin):
-  plugin_name = "My Awesome Plugin"
+  plugin_name = "My_Awesome_Plugin"
```

Comment:
OK: Plugin names are also conventionally lowercase (as they’re internal
keys that appear in URLs, and their capitalization is not user-facing),
but it’s fine with me if we change that in a separate PR.

Reply:
Done in this PR.  Thanks for the review!